### PR TITLE
Remove DS dependencies

### LINKED ISSUE #1859: Autoupdate problem after deleting a motion poll
motion detail view (meta box) is not updated if motion poll is deleted

### PR DESCRIPTION
**Pull request by ostcar**
Please test this. It works on motion polls.

fixes #1859

### CODE REVIEW

**Comment by emanuelschuetze** [comment]
Tested: Works as expected.

File: bower.json [hunk 1]
```diff
@@ -32,5 +32,8 @@
     "angular-ckeditor": "~1.0.0",
     "roboto-condensed": "~0.3.0",
     "open-sans-fontface": "https://github.com/OpenSlides/open-sans.git#1.4.2.post1"
+  },
+  "resolutions": {
+    "angular": "^1.2.x"
```
Comment (normanjaeckel) [inline bower.json]: I do not understand why to use this fix old version.

Comment (emanuelschuetze) [inline bower.json]: That will be override by #1903. angular-pdf needs angular 1.4.8+. So merge this here.

Comment (normanjaeckel) [inline bower.json]: Ok. You promise it ... :-)

**Pull request merged**